TypeCobol.LanguageServer.Test/LSRTestHelper.cs

```diff
@@ -86,6 +86,9 @@ public static void Test(string testFolderName, LsrTestingOptions lsrTestingOptio
             //Specify log file for TC LSP
             var tcLogFile = Path.Combine(workingDir.FullName, "TCLSPLog.txt");
 
+            //Check Cpy Copy names file
+            FileInfo cpyCopiesFile = new FileInfo(Path.Combine(testWorkingDirectory, "input", "CpyCopies.lst"));
```

Comment:
No need to create a `FileInfo` instance, `File.Exists` will do the job.

Reply:
But it's for getting the absolute file path also .